static/css/style.css

```diff
@@ -3702,3 +3830,81 @@ viewLog.mako
 .log-filter {
     margin-top: 20px;
 }
+
+/* ========================================================================
+css helper classes
+===========================================================================*/
+/* Some css helper classes */
+
+.top-5 {
+    margin-top: 5px;
+}
+
+.top-10 {
+    margin-top: 10px;
+}
+
+.top-15 {
+    margin-top: 15px;
+}
+
+.top-20 {
+    margin-top: 20px;
+}
+
+.top-25 {
+    margin-top: 25px;
+}
+
+.bottom-5 {
+    margin-bottom: 5px;
+}
+
+.bottom-10 {
+    margin-bottom: 10px;
+}
+
+.bottom-15 {
+    margin-bottom: 15px;
+}
+
+.bottom-20 {
+    margin-bottom: 20px;
+}
+
+.bottom-25 {
+    margin-bottom: 25px;
+}
+
+/* Submenu */
+
+@media screen and (min-width: 0px) and (max-width: 768px) {
+  .full-width { display: none; }  /* show it on small screens */
+  .mobile { display: block;}
+  .mobile .ui-icon {display: inline-block;}
+  .dropdown-menu-custom > li > a {
+      padding: 15px 36px 15px 5px;
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+  }
+  .mobile > ul > li > a {
+      padding-top: 15px;
+      padding-bottom: 15px;
+      padding-left: 8px;
+  }
+  .mobile .img-align {
+      width: 25px;
+      float: left;
+  }
+}
+
+#SubMenu > a {
```

Comment:
We should add a standard to only use lowercase along with hyphens for class names and IDs.